notebooks/01_Jupyter_Notebook/01_01_Notebook_Development.ipynb

```diff
@@ -1,23 +1,5 @@
 {
  "cells": [
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Install Python packages in the current Jupyter Kernel\n",
-    "\n",
-    "If you're in the jupyter notebook and you want to install a package with `pip`, you might be tempted to use the `!` notation to run pip directly as a shell command from the notebook."
```

Comment:
Any reason we delete this paragraph?

Reply:
Okay, it's deleted because we don't use any `!pip` cmd in the notebook. Will fix it in the new  version.

Reply:
Please keep it here. I agree we don't use in this notebook, in other notebook, we still use it. As this is the introduction, we want some of the users understand that.

Reply:
You're right. I fixed it in the latest version.